lib/msal-browser/src/broker/nativeBroker/PlatformAuthProvider.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    LoggerOptions,
+    IPerformanceClient,
+    LogLevel,
+    Logger,
+    AuthenticationScheme,
+} from "@azure/msal-common/browser";
+import { name, version } from "../../packageMetadata.js";
+import {
+    BrowserConfiguration,
+    DEFAULT_NATIVE_BROKER_HANDSHAKE_TIMEOUT_MS,
+} from "../../config/Configuration.js";
+import { BrowserPerformanceClient } from "../../telemetry/BrowserPerformanceClient.js";
+import { NativeMessageHandler } from "./NativeMessageHandler.js";
+import { NativeConstants } from "../../utils/BrowserConstants.js";
+import { PlatformDOMHandler } from "./PlatformDOMHandler.js";
+
+export const PLATFORM_EXTENSION_PROVIDER = "NativeMessageHandler";
+export const PLATFORM_DOM_PROVIDER = "PlatformDOMHandler";
+
+export async function isPlatformBrokerAvailable(
+    loggerOptions?: LoggerOptions,
+    perfClient?: IPerformanceClient
+): Promise<boolean> {
+    const defaultLoggerOptions: LoggerOptions = {
+        loggerCallback: (): void => {
+            // Empty logger callback
+        },
+        piiLoggingEnabled: false,
+        logLevel: LogLevel.Trace,
+    };
+
+    const logger = new Logger(
+        loggerOptions || defaultLoggerOptions,
```

Comment:
Logger will inject its own default
```suggestion
        loggerOptions || {},
```